### ISSUE TITLE
Search doesn't work

### ISSUE THREAD

**Issue by gonsolo** [komoot/photon#600]
I tried to search for two Bavarian cities and photon doesn't find them (Ingolstadt, Rosenheim). It works on komoot.de but doesn"t at https://photon.komoot.io. See the attached pictures.

![ingolstadt_komoot](https://user-images.githubusercontent.com/2041764/127634081-75263aad-80c6-4991-8f81-10f228a44916.png)
![ingolstadt_photon](https://user-images.githubusercontent.com/2041764/127634090-4bae3c6b-899f-4ffa-9905-ae5f555efe20.png)

**Comment by lonvia** [comment]
Looks like we need to tweak the default location bias scale some more. Search without location bias works fine: https://photon.komoot.io/api/?q=ingolstadt

Even a mild increase of location_scale_bias works wonders: https://photon.komoot.io/api/?q=ingolstadt&limit=5&lat=52.387963405382365&lon=13.058280944824219&debug=1&location_bias_scale=0.21

The search on the commercial Komoot website is unrelated.

**Comment by karussell** [comment]
Another example are the tests in [our converter](https://github.com/graphhopper/geocoder-converter/blob/master/src/test/java/com/graphhopper/converter/resource/ConverterResourcePhotonTest.java#L58):

https://photon.komoot.io/api/?q=beer&lat=48.774675&lon=9.172136&location_bias_scale=0.1

=> the result is fine

but now when increasing the location bias I was hoping to get a result closer to the specified coordinate but the opposite happens:

https://photon.komoot.io/api/?q=beer&lat=48.774675&lon=9.172136&location_bias_scale=0.5

**Comment by lonvia** [comment]
Yeah, it might have been better to switch that around. The given weight is applied to the importance. Too late to change now.

**Comment by karussell** [comment]
What if we deprecate this parameter and introduce a new one like `location_bias` with the more intuitive meaning?
